ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/TransactionProcessor.java

```diff
@@ -95,6 +97,13 @@ default boolean isSuccessful() {
      * @return the validation result, with the reason for failure (if applicable.)
      */
     ValidationResult<TransactionInvalidReason> getValidationResult();
+
+    /**
+     * Returns the reason a transaction was reverted (if applicable).
```

Comment:
Missing word.

`Returns the reason why a transaction was reverted (if applicable).`

Reply:
done